README.md

```diff
@@ -176,7 +176,7 @@ Run `npm run build` to compile the code to Javascript.
 
 Run `node dist/examples/runFasterCode.js` (to use GraphQL) or `node dist/examples/runOldCode.ts` (to use REST calls), to run the example program. It requires internet access, since it calls the GitHub API. It will take a couple minutes to complete. Some of the output includes the word "ERROR", so don't panic.
 
-Husky now runs linting checks pre-commit. Either manually fix the errors or run `npm run lint:fix` to automatically fix any errors.
+Ensure to lint your code by running `npm run lint` before submitting any code for review. Either manually fix the errors or run `npm run lint:fix` to automatically fix any errors. Husky set up to run linting checks pre-commit which should prevent being able to commit linting errors; however, There is an [bug](https://github.com/typicode/husky/issues/639) in husky where occasionally the hooks won't run in an IDE.
```

Comment:
@redgardner  This looks great. Just a couple minor requests:
- I think the third sentence should say "Husky is set up" (insert the "is").
- It should probably be "a [bug]" rather than "an [bug]".
- In markdown, I have recently developed a preference to split a long paragraph like this into multiple lines, to simplify future diffs. (If that's too much of a pain, I'm wiling to merge it with the current formatting.)

Thanks!

Reply:
🤦  Sorry for the bad grammar this morning. All filed up and I think the split paragraphs looks nicer now